src/append/opentelemetry.rs

```diff
@@ -80,16 +80,37 @@ impl OpentelemetryLogBuilder {
     }
 
     /// Add a label to the resource.
-    pub fn add_label(mut self, key: impl Into<Cow<'static, str>>, value: impl Into<Cow<'static, str>>) -> Self {
+    pub fn with_label(
```

Comment:
IIRC in builder pattern, `add_` means add on current state, while `with_` means replace current state.

So I'd prefer `add_label` and `add_labels`.

Reply:
but `add_xxx` is rust usually means `fn add_xxx(&mut self, x)`

Reply:
https://users.rust-lang.org/t/next-steps-in-builder-pattern/90341 adopts an `add_` flavor.

While [`derive_builder`](https://crates.io/crates/derive_builder) use `xxx` directly without any prefix. That looks reasonable to me too. And thus:

```rust
pub fn label(mut self, key, value);
pub fn labels(mut self, labels: impl ...);
```

Reply:
Updated.